core/domain/exp_services.py

```diff
@@ -1124,6 +1456,21 @@ def get_next_page_of_all_non_private_commits(
     fetch_page() at:
 
         https://developers.google.com/appengine/docs/python/ndb/queryclass
+
+    Args:
+        page_size: int. Number of commits that are in the commit list page.
+        urlsafe_start_cursor: str. If this is not None, then the returned
+            commits start from cursor location. Otherwise they start from the
+            beginning of the list of commits.
+
+    Returns:
+        tuple. A 2-tuple consisting of:
+            -list(ExplorationCommitLogEntry). A list containing
+                ExplorationCommitlogEntry domain object
```

Comment:
even in docstring tuple was misspelled with triple *The return value is a triple** Done! :) Thanks.